src/Api/Email.php

```diff
@@ -31,29 +30,54 @@ public function __construct(UniOneClient $client)
 
   /**
    * Send an email.
-   * @param  EmailData       $mail the request parameters
+   *
+   * @param array {
+   *                'recipients': array {array {'email': string}},
+   *                'body': array,
+   *                'subject': string,
+   *                'from_email': string,
+   *             } $params the request parameters
+   * @param  array           $headers the request headers
    * @return array           the response with the status code
    * @throws GuzzleException
+   * @see  https://docs.unione.io/en/web-api-ref#email-send Documentation of unione email-send
    */
-  public function send(EmailData $mail): array
+  public function send(array $params, array $headers = []): array
   {
-      $path = 'email/send.json';
-      $headers = $mail->getRequestHeaders();
-      $body = $mail->toArray();
+      if (!empty($params['message'])) {
+          $params = $params['message'];
+      }
 
-      return $this->client->httpRequest($path, $body, $headers);
+      Assert::isArray($params['recipients'], 'The recipients params must be an array. Got: %s');
+      Assert::isArray($params['body'], 'The body params must be an array. Got: %s');
+      Assert::string($params['subject'], 'The subject must be a string. Got: %s');
+      Assert::email($params['from_email'], 'The from_email must be an email. Got: %s');
+
+      if (!empty($params['recipients'])) {
+          foreach ($params['recipients'] as $item) {
+              Assert::email($item['email'], 'The email must be an email. Got: %s');
```

Comment:
"The email must be an email" looks confusing.

How about "Recipient should be an array with 'email' key containing a valid email address."